### PR TITLE
feat(groups): support command and <<

### PR DESCRIPTION
**Pull request by boc-tothefuture**
This addresses #180 

We have a broader #202 to address other group issues, I would like to have this one reviewed and released since a user was asking for it while we work through the broader strategic group issues.

### CODE REVIEW

File: features/groups.feature [hunk 1]
```diff
@@ -93,3 +93,17 @@ Feature:  groups
...
+      """
+        GroundFloor.<method> <value>
+      """
+    When I deploy the rules file
+    Then "Bedroom_Temp" should be in state "<value>" within 5 seconds
+    And "Den_Temp" should be in state "<value>" within 5 seconds
+    Examples:
+      | method  | value |
+      | <<      | 55    |
+      | command | 55    |
```
Comment (jimtng) [inline features/groups.feature:108]:
```suggestion
      | command | 60    |
```
So that we know it wasn't the value from before

Comment (boc-tothefuture) [reply features/groups.feature:108]: Can do..  but the cucumber framework wipes out all items and rules between runs.  

But doenst' hurt either way.

**Comment by jimtng** [comment]
LGTM

**Review by pacive — APPROVED** [review]
Looks good

**Pull request merged**